test/utility.hpp

```diff
@@ -48,22 +48,27 @@
 #define TEST_NAME \
     std::string(boost::unit_test::framework::current_test_case().p_name)
 
-#define START_BLOCKCHAIN(name, flush) \
-    threadpool pool; \
-    database::settings database_settings; \
-    database_settings.flush_writes = flush; \
-    database_settings.directory = TEST_NAME; \
-    BOOST_REQUIRE(test::create_database(database_settings)); \
-    blockchain::settings blockchain_settings; \
-    block_chain name(pool, blockchain_settings, database_settings); \
+#define START_BLOCKCHAIN(name, flush, catalog)                                 \
```

Comment:
Avoid the `\` spaces. it makes keeping them lined up a hassle. See misalignment below.